server/fishtest/static/css/live_elo.css

```diff
@@ -1,30 +0,0 @@
-div.error {
-  display: none;
-}
-
-span.pointer {
-  cursor: default;
-  -webkit-user-select: none;
-  -khtml-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  -o-user-select: none;
-  user-select: none;
-}
-
-a {
-  color: #409e42;
-  text-decoration: none;
-}
-
-a:hover {
-  color: #5f9960;
-}
-
-a:focus {
-  color: #497449;
-}
-
-td:last-child {
-  word-break: break-word;
-}
```

Comment:
Looks like removing this is adding a very small horizontal scroll on my phone

Reply:
I find horizontal scrolling rather annoying. Perhaps we could move this to application.css?

Reply:
Confirmed. Embarrassedly I haven't idea how to fix...

Reply:
I added a style tag in the mako template to break word for the last child, it works.